docs/api/segment-content.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "ion-segment-content"
+---
+
+<head>
```

Comment:
I think we should still include Props & Events for this component, it will just show up as none available: https://ionicframework.com/docs/api/app#properties

Reply:
I agree, makes it look consistent.

Reply:
The sidebar will automatically show up once there are page headers